src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/ApplicationMetadata.cs

```diff
@@ -1,64 +1,65 @@
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
 using System.Linq;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
 
 namespace Altinn.Platform.Storage.Models
 {
     /// <summary>
-    /// Model for application information
+    /// Model for application metadata.
     /// </summary>
     [Serializable]
     public class ApplicationMetadata
     {
         /// <summary>
-        /// unique id
+        /// Unique id of the application, e.g. TEST-sailor.
         /// </summary>
         [JsonProperty(PropertyName = "id")]
         public string Id { get; set; }
 
         /// <summary>
-        /// id of the application, e.g. KNS/sailor
+        /// the application build version 
         /// </summary>
-        [JsonProperty(PropertyName = "applicationId")]
-        public string ApplicationId { get; set; }
+        [JsonProperty(PropertyName = "versionId")]
```

Comment:
Will VersionId be generated automatically on deployment?

Reply:
Version id must be provided by the deploy process. currently it is the commit id of the gitea.